selfdrive/car/card.py

```diff
@@ -222,6 +231,12 @@ def state_publish(self, CS: car.CarState):
     cs_send.carState.cumLagMs = -self.rk.remaining * 1000.
     self.pm.send('carState', cs_send)
 
+    if RD is not None:
+      tracks_msg = messaging.new_message('liveTracks')
+      tracks_msg.valid = CS.canValid and len(RD.errors) == 0
```

Comment:
why is the carState canValid relevant here? if the radar bus is good, then the tracks should be valid